vllm/lora/fully_sharded_layers.py

```diff
@@ -88,14 +92,14 @@ def can_replace_layer(cls, source_layer: nn.Module,
         )
 
 
-def _mcp_apply(x, bias, layer):
+def _mcp_apply(x, bias, layer: QKVParallelLinearWithLora):
     """
-    MergedColumnParallelLinearWithShardedLoRA and 
-    MergedQKVParallelLinearWithShardedLora share the same 
+    MergedColumnParallelLinearWithShardedLoRA and
+    QKVParallelLinearWithShardedLora share the same
     LoRa weight application method.
     
     The main difference is the step by shard_size for lora_b which can
-    vary for MergedQKVParallelLinearWithShardedLora but is constant for 
+    vary for MergedQKVParallelLinearWithShardedLora but is constant for
```

Comment:
nit: this should be `QKVParallelLinearWithShardedLora`?

Reply:
This is `MergedQKVParallelLinearWithShardedLora`, but indeed there was an issue with this docstring. I have already fixed it. Thanks